packages/ses/src/whitelist-intrinsics.js

```diff
@@ -86,9 +72,48 @@ export default function whitelistIntrinsics(
   intrinsics,
   markVirtualizedNativeFunction,
 ) {
-  // These primities are allowed allowed for permits.
+  // These primitives are allowed allowed for permits.
   const primitives = ['undefined', 'boolean', 'number', 'string', 'symbol'];
 
+  // These symbols are allowed as well-known symbols
+  const wellKnownSymbolNames = new Map(
+    intrinsics.Symbol
+      ? arrayMap(
+          arrayFilter(
+            entries(whitelist.Symbol),
+            ([name, permit]) =>
+              permit === 'symbol' &&
+              typeof intrinsics.Symbol[name] === 'symbol',
+          ),
+          ([name]) => [intrinsics.Symbol[name], `@@${name}`],
+        )
+      : [],
+  );
+
+  /**
+   * asStringPropertyName()
+   *
+   * @param {string} path
+   * @param {string | symbol} prop
+   */
+  function asStringPropertyName(path, prop) {
+    if (typeof prop === 'string') {
+      return prop;
+    }
+
+    const wellKnownSymbol = mapGet(wellKnownSymbolNames, prop);
+
+    if (typeof prop === 'symbol') {
+      if (wellKnownSymbol) {
+        return wellKnownSymbol;
+      } else {
+        return `Unique${String(prop)}`;
```

Comment:
It would be nice to document the nature of `UniqueSymbol(…)` (and string representation of property keys in general) somewhere, although I'm not sure where.

Reply:
done in https://github.com/endojs/endo/commit/6f6fffb3875fb91708996333f51647a6a447dfad#diff-a241982c4634b07275319f160e1d3cb9b89929217019708fae1b5ea3bb4e8e77